### PR TITLE
Welcome message + camelot -> jesters

### PR DESCRIPTION
**Pull request by ghost**
- Horace now sends a welcome DM to new users upon accepting the TOS. 
- Info terms/invites both reflect name change of Camelot server to Jesters of Academia. 

(Probably should have made two PRs but this get it out quicker 😆)

### CODE REVIEW

File: commands/invite.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ module.exports.execute = async (client, message, args) => {
 	let inviteText = 'Invite to the KOA server: https://discord.gg/EYX7XGG';
 
 	if(args[0]) {
-		if(args[0].toLowerCase() === 'camelot') {
-			inviteText = 'Invite to the Knights of Academia: Camelot server: https://discord.gg/wu3a6JA';
+		if(args[0].toLowerCase() === 'jesters') {
```
Comment (Tekrus) [inline commands/invite.js]: I think we should add a case for camelot still, while people are getting used to it, so we have:

camelot/jesters/joa

File: data/infoTerms.js [hunk 2]
```diff
@@ -65,9 +65,9 @@ module.exports = [
 		description: 'The Book of the Month (BOTM) activity involves our BOTM Moderator posting polls for people to vote on what they want the next month\'s book to be, that every member of the club will endeavour to read over the course of the month. Look forward to interesting discussion and listening to other people\'s perspectives on the book! Feel free to check out <#382510727614824448> for more info.'
 	},
 	{
-		displayname: 'Camelot',
-		names: ['camelot'],
-		description: 'The place for memes, ranting, debates, and pretty much anything else that might not be allowed in KOA Main. Feel free to grab a link to it with `!invite camelot` or in <#403260793644384266>!'
+		displayname: 'Jesters of Academia',
+		names: ['jesters', 'jesters-of-academia'],
```
Comment (Tekrus) [inline data/infoTerms.js]:
```suggestion
		names: ['jesters', 'jesters-of-academia', 'camelot'],
```

File: commands/invite.js (hunk 1 above)
Comment (ghost) [reply commands/invite.js]: Will add this in tonight for invite, just committed your suggestion on the info.

**Review by Dendrobyte — APPROVED** [review]
Grammar 👏 approved 👏

**Comment by ghost** [comment]
Fixed problems in Dan's review. Didn't add in a test for camelot arg seeing as it might be removed an update from now. 

Fixes #131 

Merging now.

**Pull request merged**